src/inline-assembly.md

```diff
@@ -477,3 +478,135 @@ To avoid undefined behavior, these rules must be followed when using function-sc
   - The compiler is currently unable to detect this due to the way inline assembly is compiled, but may catch and reject this in the future.
 
 > **Note**: As a general rule, the flags covered by `preserves_flags` are those which are *not* preserved when performing a function call.
+
+### Directives Support
+
+Inline ASM supports a subset of the directives supported by both GNU AS and LLVM's internal assembler, given as follows.
+The result of using other directives is assembler-specific (and may cause an error, or may be accepted as-is).
+
+The following directives are guaranteed to be supported by the assembler:
+
+
+- `.2byte`
```

Comment:
Could you please either sort the list, or group the list into logical groups?

It may make some sense to group things like `byte`, `short`, `word`, `quad`, and similar together, but that seems like substantially more organizational work, and in practice I would expect that a fully sorted list may be easier to use as a reference.